# Generated by Django 3.0.7 on 2020-06-09 16:10

from django.conf import settings
from django.db import migrations, models
import django.db.models.deletion


class Migration(migrations.Migration):

    dependencies = [
        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
        ('cont', '0009_auto_20200408_0831'),
    ]

    operations = [
        migrations.AlterField(
            model_name='modelo',
            name='nome',
            field=models.CharField(max_length=255, unique=True),
        ),
        migrations.CreateModel(
            name='OntDefeitoDevolucao',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('data', models.DateTimeField(auto_now=True)),
                ('defeito', models.ForeignKey(default=None, on_delete=django.db.models.deletion.CASCADE, related_name='devolucao_defeito', to='cont.OntSaida')),
                ('fornecedor', models.ForeignKey(on_delete=django.db.models.deletion.PROTECT, related_name='devolucao_fornecedor', to='cont.Cliente')),
                ('ont', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='devolucao_ont', to='cont.Ont')),
                ('user', models.ForeignKey(on_delete=django.db.models.deletion.PROTECT, related_name='devolucao_user', to=settings.AUTH_USER_MODEL)),
            ],
        ),
        migrations.CreateModel(
            name='OntDefeito',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('data', models.DateTimeField(auto_now=True)),
                ('ont', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='defeito_ont', to='cont.Ont')),
                ('user', models.ForeignKey(on_delete=django.db.models.deletion.PROTECT, related_name='dfeito_user', to=settings.AUTH_USER_MODEL)),
            ],
        ),
    ]
